solidity/test/BancorFormula.js

```diff
@@ -5,7 +5,6 @@ const { expectRevert, BN } = require('@openzeppelin/test-helpers');
 // decimal numbers.
 const BigNumber = require('bignumber.js');
 const Decimal = require('decimal.js');
-Decimal.set({ precision: 100, rounding: Decimal.ROUND_DOWN });
```

Comment:
Where did this go?

Reply:
In a single location in `truffle-config.js`

Reply:
Cool!!! Always wanted to do that, just didn't think it was feasible.
Now that I think about it, `truffle-config.js` is... well, JS, so of course it's feasible!